web/template/unlink_person.gohtml

```diff
@@ -0,0 +1,96 @@
+{{ template "page" . }}
+
+{{ define "title" }}Unlink Record{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            {{ if .Success }}
+                {{ template "success-banner" "You have successfully unlinked the record(s)." }}
+            {{ end }}
+
+            <h1 class="govuk-heading-l app-!-embedded-hide">Unlink Record</h1>
+
+                <table class="govuk-table">
+                    <caption class="govuk-table__caption govuk-table__caption--m">Primary record</caption>
+                    <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Name</th>
+                        <td class="govuk-table__cell">{{ .Person.Firstname }} {{ .Person.Surname }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Contact ref #</th>
+                        <td class="govuk-table__cell">{{ .Person.UID }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Address</th>
+                        <td class="govuk-table__cell">{{ .Person.AddressLine1 }}</td>
+                    </tr>
```

Comment:
Not a blocked, but in case it comes up: the old forms didn't ever actually show the address because they referred to an undefined variable called `uiAddress`